circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/aws/JceksAmazonS3ClientFactory.java

```diff
@@ -136,10 +136,13 @@ private AmazonS3 newInstance(
 
   private HadoopAWSCredentialProviderChain getCredentialsProviderChain(String assumedRole) {
     if (assumedRole != null) {
+      LOG.info("Creating credential chain for assuming role {}", assumedRole);
       return new HadoopAWSCredentialProviderChain(createNewConf(conf, assumedRole));
     } else if (security.getCredentialProvider() != null) {
+      LOG.info("Creating credential chain for assuming role with cred provider {}", security.getCredentialProvider());
```

Comment:
comment is wrong here - this case is for creating a credential provider using Jceks.